src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+                    System.out.println( number + "." + todos[i]);
+                }
+                System.out.println("____________________________________________________________");
+
+            } else {
+                String[] arr = text.split(" ", 2);
+                String action = arr[0];
+                String input = arr[1];
+
+                if (action.equals("done")){
+                    int setAsDone =Integer.parseInt(input) ;
+                    setAsDone--;
+                    todos[setAsDone].setDone(true);
+
+                    System.out.println("____________________________________________________________");
+                    System.out.println(todos[setAsDone]);
+                    System.out.println("____________________________________________________________");
+
+                } else if (action.equals("todo")) {
+                    Todo newTodo = new Todo(input);
+                    todos[index++] = newTodo;
+
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println(todos[index - 1]);
+                    System.out.println("Now you have " + index + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+
+                } else if (action.equals("deadline")) {
+                    String[] inputs = input.split(" /by ", 2);
+                    Deadline newDeadline = new Deadline(inputs[0], inputs[1]);
+                    todos[index++] = newDeadline;
+
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Got it. I've added this task: " + input);
+                    System.out.println(todos[index - 1]);
+                    System.out.println("Now you have " + index + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+
+                } else if (action.equals("event")) {
+                    String[] inputs = input.split(" /at ", 2);
+                    Event newEvent = new Event(inputs[0], inputs[1]);
+                    todos[index++] = newEvent;
+
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Got it. I've added this task: " + input);
+                    System.out.println(todos[index - 1]);
+                    System.out.println("Now you have " + index + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+
+                } else {
+                    System.out.println("Sorry, I do not know what you mean.");
+                }
+            }
+
+        }
+
+        scanner.close();
+
     }
 }
```

Comment:
The main method contains almost all the operations of the program. Please consider using methods for SLAP.